core/templates/domain/statistics/learner-action.model.ts

```diff
@@ -105,48 +102,32 @@ export type LearnerAction = (
   AnswerSubmitLearnerAction |
```

Comment:
Hey @brianrodri `LearnerAction` has already been defined as a type so I wasn't able to give the class same name ,that's why I made it `LearnerActionModel`

Reply:
Hmm I see, in that case I think I'd rather refactor these classes to fit the pattern rather than break our conventions. Sorry, but could you please skip this file while I look into rewriting it?

Reply:
Sure @brianrodri !Thanks